src/redux/books/booksSlice.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-param-reassign */
 import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
 import axios from 'axios';
 import API_URL from '../../globals';
```

Comment:
I'm unable to open your app in the browser, please kindly fix this issue so that your app opens in the browser without any error as mentioned by the previous reviewer, 👍🛠

![image](https://github.com/kazaneza/Bookstore/assets/78251927/2d3b3955-185e-48af-b1cd-67a1b7b2624b)

You can check this [thread](https://stackoverflow.com/questions/69826609/sass-error-import-variables-sasserror-cant-find-stylesheet-to-import) for a fix 🤗